block/manager.go

```diff
@@ -119,6 +138,11 @@ func NewManager(
 		conf.DABlockTime = defaultDABlockTime
 	}
 
+	if conf.BlockTime == 0 {
+		logger.Info("WARNING: using default block time", "BlockTime", defaultBlockTime)
```

Comment:
[non-blocking] I wouldn't add `WARNING` to an info log. I think people will be confused and think something is wrong.

Reply:
Makes sense, removed this.